packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -7413,6 +7429,10 @@ function mockLastDeploymentRequest() {
 	msw.use(...mswSuccessLastDeployment);
 }
 
+interface Dictionary<T> {
+	[Key: string]: T;
+}
+
```

Comment:
I don't think this is really needed. It is easy enough to use the standard `Record` type:

```ts
expectedUnsafeMetadata?: Record<string, string>
```
```